dev/PushNotifications/PushNotificationTelemetry.h

```diff
@@ -117,8 +117,9 @@ class PushNotificationTelemetry : public wil::TraceLoggingProvider
     }
     CATCH_LOG()
 
-        DEFINE_EVENT_METHOD(LogInvokeAll)(
-            winrt::hresult hr) noexcept try
+    DEFINE_EVENT_METHOD(LogInvokeAll)(
```

Comment:
Will this become an activity once the PR to move push notifications from events to activities is merged?

Reply:
Yes. I think that's one of the reasons this PR is still a draft 😅.

Reply:
I merged against the latest develop commit, which has the activities I introduced.